### ISSUE TITLE
installer: Unclear how scripts/binaries are generated, how to file bug reports against installer code?

### ISSUE THREAD

**Issue by EricCousineau-TRI** [conda/conda#10470]
### TL;DR
Where's the source for the installer? When looking at it, it seems like there's some dead code that's awkardly confusing?

See:
https://gist.github.com/EricCousineau-TRI/cc7cf8284ed90a29821f85b72cfa319c

### Background
Was doing basic investigation as a user for https://github.com/jupyterlab/jupyterlab/issues/8959, and wanted to do some dev-level mods, so used miniconda per the JupyterLab contributor suggestions.

(To clarify, it greatly simplified me setting up an env to work with it - so awesome overall! This is me just splitting hairs :P)

I had looked at the MiniConda distribution's installer; I don't love that it strongly suggests modifying `~/.bashrc` / should run `conda init` (which does the same thing), but I can understand the main target audience generally doesn't care.
(FWIW I'm picky because it doesn't play well with distributed builds of our software (https://github.com/RobotLocomotion/drake/pull/10961); sure, we could support it, but we haven't reached that point yet (https://github.com/RobotLocomotion/drake/issues/13945)).

In doing an install on my system, I decided to do a more contained/non-invasive installation:
https://gist.github.com/EricCousineau-TRI/b713101b2a90d16e230fc250077ec4a2

I then wanted to PR this to the Conda docs, and did notice the "silent mode" installation:
https://conda.io/projects/conda/en/latest/user-guide/install/macos.html#install-macos-silent

But when comparing those docs versus the code, some stuff seems to not have full parity? (also, some other minor shell bugs :shrug:)

### Mebbe Bugs?
The `-s` option is documented, but `SKIP_SCRIPTS` doesn't do anything?
https://gist.github.com/EricCousineau-TRI/cc7cf8284ed90a29821f85b72cfa319c#file-miniconda3-latest-linux-x86_64-part-sh-L108

`BASH_RC` is declared, but ain't used? (at least not in readable source code?)
https://gist.github.com/EricCousineau-TRI/cc7cf8284ed90a29821f85b72cfa319c#file-miniconda3-latest-linux-x86_64-part-sh-L471

The `-b` option doesn't mention that it kinda covers the `-s` option?
https://gist.github.com/EricCousineau-TRI/cc7cf8284ed90a29821f85b72cfa319c#file-miniconda3-latest-linux-x86_64-part-sh-L69-L70
